specs/deneb/p2p-interface.md

```diff
@@ -292,6 +292,10 @@ Clients MUST support requesting sidecars since `minimum_request_epoch`, where `m
 Clients MUST respond with at least one sidecar, if they have it.
 Clients MAY limit the number of blocks and sidecars in the response.
 
+In the event of equivocating blobs, clients MUST serve only blobs with matching KZG commitments if they have seen the block.
```

Comment:
I'm not sure this is necessary  if #3531 is merged. Because you now have an explicit tie between the blobs and the block header at the point of gossip acceptable